npm/angular/.eslintrc

```diff
@@ -1,8 +1,7 @@
 {
   "plugins": [
     "cypress",
-    "@cypress/dev",
-    "@angular-eslint/eslint-plugin"
```

Comment:
I removed this because running lint was throwing an error saying it couldn't find this deep. It's not really providing any value currently and we can always add in these or other angular eslint rules in the future